specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2019-08-01/databoxedge.json

```diff
@@ -0,0 +1,4919 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/{deviceName}/storageAccounts/{storageAccountName}/containers": {
+      "get": {
+        "tags": [ "Containers" ],
+        "summary": "Lists all the containers of a storage Account in a Data Box Edge/Data Box Gateway device.",
+        "operationId": "Containers_ListByStorageAccount",
+        "parameters": [
+          {
+            "name": "deviceName",
+            "in": "path",
+            "description": "The device name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "storageAccountName",
+            "in": "path",
+            "description": "The storage Account name.",
+            "required": true,
+            "type": "string"
+          },
+          { "$ref": "#/parameters/subscriptionIdParameter" },
+          { "$ref": "#/parameters/resourceGroupNameParameter" },
+          { "$ref": "#/parameters/apiVersionParameter" }
+        ],
+        "responses": {
+          "200": {
+            "description": "The collection of all the containers on the device.",
+            "schema": { "$ref": "#/definitions/ContainerList" }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": { "$ref": "#/definitions/CloudError" }
+          }
+        },
+        "x-ms-pageable": { "nextLinkName": "nextLink" },
+        "x-ms-examples": {
+          "ContainerListAllInDevice": {
+            "$ref": "./examples/ContainerListAllInDevice.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/{deviceName}/storageAccounts/{storageAccountName}/containers/{containerName}": {
+      "get": {
+        "tags": [ "Containers" ],
+        "summary": "Gets a container by name.",
+        "operationId": "Containers_Get",
+        "parameters": [
+          {
+            "name": "deviceName",
+            "in": "path",
+            "description": "The device name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "storageAccountName",
+            "in": "path",
+            "description": "The Storage Account Name",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "containerName",
+            "in": "path",
+            "description": "The container Name",
+            "required": true,
+            "type": "string"
+          },
+          { "$ref": "#/parameters/subscriptionIdParameter" },
+          { "$ref": "#/parameters/resourceGroupNameParameter" },
+          { "$ref": "#/parameters/apiVersionParameter" }
+        ],
+        "responses": {
+          "20
```

Comment:
Can this be modeled? From an SDK/Documentation perspective this should be more customer focused.